### PR TITLE
Create onboarding activity

### PR DESCRIPTION
**Pull request by devKate**

### CODE REVIEW

File: app/src/main/java/com/citylights/chatapp/ui/onboarding/OnboardingActivity.kt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_onboarding)
+
+        viewPagerOnboarding.adapter = OnboardingPagerAdapter(supportFragmentManager)
+
+        buttonBackOnboarding.setOnClickListener { viewPagerOnboarding.setCurrentItem(viewPagerOnboarding.getCurrentItem() - 1) }
+        buttonNextOnboarding.setOnClickListener { viewPagerOnboarding.setCurrentItem(viewPagerOnboarding.getCurrentItem() + 1) }
+        // TODO: build ProfileActivity
+//        buttonSkipOnboarding.setOnClickListener { startActivity(Intent(this, ProfileActivity::class.java)) }
```
Comment (nishtahir) [inline app/src/main/java/com/citylights/chatapp/ui/onboarding/OnboardingActivity.kt]: Not a fan of this

File: app/src/main/java/com/citylights/chatapp/ui/onboarding/OnboardingActivity.kt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ */
+class OnboardingActivity: AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_onboarding)
+
+        viewPagerOnboarding.adapter = OnboardingPagerAdapter(supportFragmentManager)
+
+        buttonBackOnboarding.setOnClickListener { viewPagerOnboarding.setCurrentItem(viewPagerOnboarding.getCurrentItem() - 1) }
```
Comment (nishtahir) [inline app/src/main/java/com/citylights/chatapp/ui/onboarding/OnboardingActivity.kt]: I'm a little worried that the lambda doesn't describe what it's supposed to do. I would pull it out into a method.

File: app/src/main/res/drawable/dot_active_onboarding.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape
+            android:innerRadius="0dp"
+            android:shape="ring"
+            android:thickness="3dp"
+            android:useLevel="false">
+            <solid android:color="@android:color/black"/>
```
Comment (nishtahir) [inline app/src/main/res/drawable/dot_active_onboarding.xml]: I'd be careful about using built in colors. Some OEMs have been known to override default color implementations.

File: app/src/main/res/layout/activity_onboarding.xml [hunk 4]
```diff
@@ -1,13 +1,60 @@
...
+        android:id="@+id/viewPagerOnboarding"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_above="@+id/bottomNavigationOnboarding"
+        android:background="@color/yellow_dark">
+
+        <android.support.design.widget.TabLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            TabLayout_tabPadding="0dp"
```
Comment (nishtahir) [inline app/src/main/res/layout/activity_onboarding.xml]: I'd extract a lot of these dimensions into dimens.

File: app/src/main/java/com/citylights/chatapp/ui/onboarding/OnboardingActivity.kt (hunk 1 above)
Comment (devKate) [reply app/src/main/java/com/citylights/chatapp/ui/onboarding/OnboardingActivity.kt]: the viewpager or the commented out code?

File: app/src/main/java/com/citylights/chatapp/ui/onboarding/OnboardingActivity.kt (hunk 2 above)
Comment (devKate) [reply app/src/main/java/com/citylights/chatapp/ui/onboarding/OnboardingActivity.kt]: ohh, okay

**Pull request merged**